script/installation/packages.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
I am not familiar with passing flags in the shebang. Does this add the intended behavior, even if both `-e` and `-x` are given?